go.mod

```diff
@@ -156,15 +156,15 @@ require (
 	github.com/prometheus/client_golang v1.19.0
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.51.1
-	github.com/redis/go-redis/v9 v9.3.0 // replaced
+	github.com/redis/go-redis/v9 v9.5.1 // replaced
```

Comment:
It's strange that this is touching a bunch of unrelated-looking dependencies, but I get the same behavior if I `go get github.com/sigstore/cosign/v2@v2.2.4` on a clean branch.